lib/presentation/ui/utils/occurrence/occurrence_location_widget.dart

```diff
@@ -115,11 +117,13 @@ class OccurrenceLocationWidget extends StatelessWidget {
             mainAxisAlignment: MainAxisAlignment.spaceEvenly,
             children: <Widget>[
               Container(
-                child: Text(
-                  "Ultima atualização : " +
-                      getFormattedDate(this._lastUpdated, FORMAT_DATE_FULL),
-                  textAlign: TextAlign.center,
-                  style: styleLastUpdated(),
+                child: Flexible(
```

Comment:
Shouldn't Flexible have always as a parent a Row/Column or a Flex Widget?

Reply:
Not having the `Flexible` widget as a direct descendant of a `Row`, `Column` or any other `Flex` widget, should break the UI.